components/script/dom/window.rs

```diff
@@ -173,21 +165,7 @@ pub struct Window {
     #[ignore_malloc_size_of = "trait objects are hard"]
     script_chan: MainThreadScriptChan,
     #[ignore_malloc_size_of = "task sources are hard"]
```

Comment:
Let's remove this annotation and add any required annotations inside of TaskManager instead.

Reply:
Sorry, I'm blocked in this I dunno how to get past: What do I have to do to have `ignore_malloc_size_of` in the `TaskManager`? I keep getting this error:

```
error[E0658]: The attribute `ignore_malloc_size_of` is currently unknown to the compiler and may have meaning added to it in the future (see issue #29642)
  --> components/script/task_manager.rs:50:5
   |
50 |     #[ignore_malloc_size_of = "task sources are hard"]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(custom_attribute)] to the crate attributes to enable
```

And I don't know how to solve it (I already tried the suggestion).

Thanks!

Reply:
Did you add `#[derive(MallocSizeOf)]` to the TaskManager struct as well?

Reply:
That work, thanks!